addons/l10n_in_withholding/__init__.py

```diff
@@ -10,15 +10,15 @@
 
 def _l10n_in_withholding_post_init(env):
     """ Existing companies that have the Indian Chart of Accounts set """
-    data = {
-        model: env['account.chart.template']._parse_csv('in', model, module='l10n_in_withholding')
-        for model in [
-            'account.account',
-            'account.tax',
-        ]
-    }
     for company in env['res.company'].search([('chart_template', '=', 'in'), ('parent_id', '=', False)]):
         _logger.info("Company %s already has the Indian localization installed, updating...", company.name)
+        data = {
+            model: env['account.chart.template']._parse_csv('in', model, module='l10n_in_withholding')
+            for model in [
+                'account.account',
+                'account.tax',
+            ]
+        }
         ChartTemplate = env['account.chart.template'].with_company(company)
```

Comment:
```suggestion
        ChartTemplate = env['account.chart.template'].with_company(company)
        data = {
            model: ChartTemplate._parse_csv('in', model, module='l10n_in_withholding')
            for model in [
                'account.account',
                'account.tax',
            ]
        }
```

And use `ChartTemplate.ref('xxx')` instead of `env.ref('account.%i_xxx')` below also

Reply:
Hello @william-andre 
Suggested changes are done.